src/main/scala/org/ergoplatform/nodeView/ErgoNodeViewHolder.scala

```diff
@@ -37,6 +39,15 @@ abstract class ErgoNodeViewHolder[State <: ErgoState[State]](settings: ErgoSetti
   override protected lazy val modifiersCache =
     new ErgoModifiersCache(settings.scorexSettings.network.maxModifiersCacheSize)
 
+  override def preStart(): Unit = {
```

Comment:
There is already a `preRestart` method in this class

Reply:
@catena2w , where? How had it compiled then?